charts/data-dashboard-backend/values.yaml

```diff
@@ -0,0 +1,118 @@
+# Default values for data-dashboard-backend.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- Number of replicas to deploy
+replicaCount: 2
+
+image:
+  # -- docker image repository
+  repository: ghcr.io/thehyve/radar-data-dashboard-backend
```

Comment:
Maybe we can separate the `registry` key?
```
image:
      registry: ghcr.io
      repository: thehyve/radar-data-dashboard-backend
```

Reply:
Yes it's a good idea. Have we moved our image hosting to Github?

Reply:
Yes, I configured this for this artifact. I do not think that this is a RADAR-base wide decision. This worked with very little effort from my side.

Reply:
If we haven't decided to move our image hosting to Dockerhub I suggest talking with Amos about how to push the `radar-data-dashboard-backend` image to radarbase dockerhub to avoid having our images scattered around in different places.

Reply:
@keyvaann Will do.

Reply:
I have pushed the _radar-data-dashboard-backend_ to RADAR-base organization. I added the _release.yaml_ workflow to GitHub actions. When the new repo receives the docker hub secrets the image will be published on dockerhub.

Reply:
Oh okay thanks. I guess we don't need the separate registry config then?